application/features/bootstrap/SyncContext.php

```diff
@@ -28,18 +29,21 @@ class SyncContext implements Context
     private $idBroker;
 
     /** @var FakeIdStore */
-    private $idStore;
+    protected $idStore;
```

Comment:
Why the visibility change?

Reply:
`NotificationContext` inherits `SyncContext` and uses `idStore` to get the user that was added earlier in the scenario. I could have changed the implementation to store the user as a property instead, but kept with the existing implementation.